index.html

```diff
@@ -6,216 +6,266 @@
-                        <div class="item">css</div>
-                        <div class="item">JavScript</div>
-                        <div class="item">html</div>
-                    </div>
+                    <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
+                    <ul class="grid-container">
+                        <li class="item">Ruby on rails</li>
+                        <li class="item">css</li>
+                        <li class="item">JavScript</li>
+                        <li class="item">html</li>
+                    </ul>
                     <button class="see-project-btn">See Project</button>
                 </div>
             </div>
             <div class="project-card">
                 <div class="descriptions">
-                    <div class="card-title"> Multi-Post Stories Gain+Glory</div>
-                    <div class="grid-container">
-                        <div class="item">Ruby on rails</div>
-                        <div class="item">css</div>
-                        <div class="item">JavScript</div>
-                        <div class="item">html</div>
-                    </div>
+                    <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
+                    <ul class="grid-container">
+                        <li class="item">Ruby on rails</li>
+                        <li class="item">css</li>
+                        <li class="item">JavScript</li>
+                        <li class="item">html</li>
+                    </ul>
                     <button class="see-project-btn">See Project</button>
                 </div>
             </div>
-
+        </div>
     </section>
     <section class="about-me">
-            <article class="about-myself">
-                <h1 style="margin: 0;">About me</h1>
-                <p>
-                    I can help you build a product , feature or website Look through some of my work and experience!
-                    If
-                    you
-                    like
-                    what you see and have a project you need coded,
-                    don't hestiate to contact me.
-                </p>
-                   <button>
-                            Get My Resume
-                   </button>
-            <img src="./img/illustration 1- about me-mobile.svg" alt="illustration-bottom-left">
-
-                   <img src="./img/illustration 2-about me mobile.svg" alt="illustration-bottom-left">
-            </article>
-
+        <article class="about-myself">
+            <h1 style="margin: 0">About me</h1>
+            <p class="refactor">
+                I can help you build a product , feature or website Look through some
+                of my work and experience! If you like what you see and have a project
+                you need coded, don't hestiate to contact me.
+            </p>
+            <button>Get My Resume</button>
+        </article>
     </section>
 
```

Comment:
- [ ] Please make sure that your mobile version stay the same. but I think there is some thing that to be improved in your mobile version design.
### Your design👇
![abtst](https://user-images.githubusercontent.com/66992901/201480990-d68d1564-97a6-4e26-b3cc-9fce0aeade4b.PNG)
### figma template👇
![abtfig](https://user-images.githubusercontent.com/66992901/201480999-0ccf8470-3f7c-4793-aef8-d4639c9a3a5a.PNG)